v2/cmd/asoctl/cmd/log.go

```diff
@@ -15,16 +15,13 @@ import (
 )
 
 var (
-	// Global logger and progress bar
-	// While these are global here, they can't be referenced from internal packages and should be passed through
-	// as parameters where needed.
-	verbose bool
+	verbose bool // True if the user specifies --verbose or -v
+	quiet   bool // True if the user specifies --quiet or -s
```

Comment:
`--quiet` == `-s` seems weird?

Why not `-q`? Or is `-s` supposed to be short for `silent` - in which case why not `--silent` to match?

Reply:
The short form is `-q`; have fixed the comment.